src/pages/Autolending/AutolendingWho.vue

```diff
@@ -304,6 +304,7 @@ export default {
 	.advanced-options-toggle {
 		color: $kiva-textlink;
 		font-weight: 300;
+		padding: 0.5rem;
```

Comment:
Unrelated change, this just makes the toggle link look better when focus style is active